agent/src/main/java/reactor/blockhound/BlockHound.java

```diff
@@ -254,6 +258,58 @@ public TypePool typePool(ClassFileLocator classFileLocator, ClassLoader classLoa
 
         private Predicate<Thread> dynamicThreadPredicate = t -> false;
 
+        /**
+         * A phase of the builder where allowed blocking methods are configured for a given class.
+         * See {@link Builder#allowBlockingCallsInside(String)}.
+         * <p>
+         * Note that methods of this class directly mutate the {@link Builder} as if one was calling
+         * {@link Builder#allowBlockingCallsInside(String, String)}. One MUST specify at least one
+         * allowed method for it to have an effect on the configuration.
+         */
+        public class BuilderAllowSpec {
+
+            final String className;
+
+            private BuilderAllowSpec(String className) {
+                this.className = className;
+            }
+
+            /**
+             * Allow blocking calls inside the static initializer block of the currently configured
+             * class. Use {@link #and()} to exit the class-specific phase and return
+             * to the {@link Builder}.
+             *
+             * @return this {@link BuilderAllowSpec} for further configuration of the given class
+             * @see #and()
+             */
+            public BuilderAllowSpec forStaticInitializer() {
+                Builder.this.allowBlockingCallsInside(this.className, "<clinit>");
+                return this;
+            }
```

Comment:
Why can't this go back to `Builder` and eliminate the need for `.and()`? Either way, as it is right now, the classname is repeated for every additional method. This would be no different. It would look like this:
```java
builder
    .allowBlockingCallsInside("...").forStaticInitializer()
    .allowBlockingCallsInside("...").forMethod("...")
   ...

```

Reply:
good point, done